sea-orm-cli/src/commands.rs

```diff
@@ -219,25 +221,37 @@ pub fn run_migrate_command(
                 false => format!("{}/", migration_dir),
             };
             println!("Initializing migration directory...");
+            let datatime = migration_name_prefix();
             macro_rules! write_file {
                 ($filename: literal) => {
                     let fn_content = |content: String| content;
                     write_file!($filename, $filename, fn_content);
                 };
                 ($filename: literal, $template: literal, $fn_content: expr) => {
-                    let filepath = [&migration_dir, $filename].join("");
+                    let filepath = [migration_dir.clone(), $filename.to_string()].join("");
+
+                    // replace path
+                    let mut handlebars = Handlebars::new();
+                    handlebars.register_template_string("path", &filepath).unwrap();
+                    let filepath = handlebars.render("path", &json!({"datatime": datatime})).unwrap();
+
                     println!("Creating file `{}`", filepath);
                     let path = Path::new(&filepath);
                     let prefix = path.parent().unwrap();
                     fs::create_dir_all(prefix).unwrap();
                     let mut file = fs::File::create(path)?;
+
+                    // replace content
                     let content = include_str!(concat!("../template/migration/", $template));
+                    handlebars.register_template_string("content", &content).unwrap();
+                    let content = handlebars.render("content", &json!({"datatime": datatime})).unwrap();
+
                     let content = $fn_content(content.to_string());
                     file.write_all(content.as_bytes())?;
                 };
             }
             write_file!("src/lib.rs");
-            write_file!("src/m20220101_000001_create_table.rs");
+            write_file!("src/{{datatime}}_create_table.rs");
```

Comment:
I think we can rename `m20220101_000001_create_table.rs` to something like `migration_template.rs`. Then, the generated could be named according to the current timestamp e.g. `mYYYYMMDD_000001_create_table.rs`